snuba/request/schema.py

```diff
@@ -104,6 +108,18 @@ def validate(self, value, dataset: Dataset, referrer: str) -> Request:
             }
 
         query = parse_query(query_body, dataset)
+        # Temporary code to collect some metrics about the usage of
+        # alias references.
+        alias_references_present = any(
+            e
+            for e in query.get_all_expressions()
+            if isinstance(e, Column) and e.alias is None
```

Comment:
```suggestion
            if isinstance(e, Column) and e.alias is not None
```

Reply:
It is actually correct this way. I am not looking for columns that define an alias. I am looking for alias references. Which, at this point of the query processing, are just columns that the parser did not assign an alias to because such alias already existed in the query.

Reply:
👍 OK thanks for clarifying.